src/CommandHandler.py

```diff
@@ -129,7 +129,28 @@ def CreateLabHandler(self, cmd: [str]):
         print('Create lab:', cmd)
 
     def AssignCourseHandler(self, cmd: [str]):
-        print('Assign course:', cmd)
+        if self.currentUser is None or not self.currentUser.RoleIn(Role.Administrator, Role.Supervisor):
+            print('Must be logged in as an Administrator or a Supervisor')
+            return
+        if len(cmd) != 2:
+            print('Invalid number of arguments')
+        c = Course(self.db)
+        c.course_name = cmd[0]
+        if not c.Exists():
+            print('This course does not exist.')
+            return
+        c.GetDetail()
+        c.instructor_email = cmd[1]
+
+        acc = Account(self.db)
+        acc.act_email = cmd[1]
+        acc.GetDetail()
```

Comment:
We should check for acc.Exists() here, like in the login command.  An exception will be thrown if we call GetDetail with an act_email that does not exist